### DISCUSSION TITLE
Convert to rem

### DISCUSSION THREAD

**Discussion by daneelf** [postcss/postcss#1671] [Q&A]
Hello everyone!

Is there a way to configure postcss (for example, in postcss.config) to convert all values in rem?

Thank you!

**Comment by ai** [comment]
I used [`postcss-pixtorem`](https://github.com/cuth/postcss-pxtorem) for this case:

```js
// postcss.config.js
module.exports = {
  plugins: [
    ['postcss-pixtorem': {
      rootValue: 16 // root font-size in your application
    }]
  ]
}
```
